gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -262,3 +258,30 @@ def validate(self, parsed_args):
             '--variant_merge_strategy {}|{}'.format(
                 MergeOptions.MOVE_TO_CALLS,
                 MergeOptions.MERGE_WITH_NON_VARIANTS))
+
+
+class PreprocessOptions(VariantTransformsOptions):
+  """Options for preprocess."""
+  _REPORT_NAME = 'conflicts_report.csv'
+  _RESOLVED_HEADERS_FILE_NAME = 'resolved_headers.vcf'
+
+  def add_arguments(self, parser):
+    parser.add_argument(
+        '--report_all',
+        type='bool', default=True, nargs='?', const=False,
+        help=('By default, only the incompatible VCF headers will be reported. '
+              'If true, it also reports the undefined headers and malformed '
+              'recorded.'))
+    parser.add_argument(
+        '--directory',
```

Comment:
I think we should ask the user to provide the full path for both the report and the resolved headers. We should make the report path required, but resolved header optional (and don't write anything if unspecified). That way, it provides more flexibility and clarity for the user and we no longer need to generate custom paths.